nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -17,7 +17,8 @@
 from .base_masker import filter_and_extract, BaseMasker
 
 
-def _iter_signals_from_spheres(seeds, niimg, radius, mask_img=None):
+def _iter_signals_from_spheres(seeds, niimg, radius, mask_img=None,
+                               allow_overlap=False):
```

Comment:
Do you thinks that this should be the default. Maybe this deserves a word in the documentation.